packages/blocks/src/root-block/edgeless/components/panel/color-panel.ts

```diff
@@ -318,6 +324,9 @@ export class EdgelessColorPanel extends LitElement {
 
   @property({ attribute: false })
   accessor value: string | null = null;
+
+  @property({ attribute: false })
+  accessor withoutTransparent: boolean = false;
```

Comment:
Negative field (`withoutSomething`) naming with falsy default value could be confusing, how about `isOpaque` / `hasBackground` / `isTransparent`?